apps/stack/template.yaml

```diff
@@ -37,6 +37,11 @@ Metadata:
           default: CloudWatch Metrics
         Parameters:
           - MetricStreamFilterUri
+      - Label:
+          default: Cloudwatch Metrics Poller Role
```

Comment:
This functionality may be useful beyond metrics collection, so maybe keep it as "Enable Observe poller"